paymentsheet/src/main/java/com/stripe/android/customersheet/injection/CustomerSheetComponent.kt

```diff
@@ -0,0 +1,26 @@
+package com.stripe.android.customersheet.injection
+
+import androidx.activity.result.ActivityResultCaller
+import androidx.lifecycle.LifecycleOwner
+import com.stripe.android.customersheet.CustomerSheet
+import com.stripe.android.customersheet.ExperimentalCustomerSheetApi
+import dagger.BindsInstance
+import dagger.Subcomponent
+
+@OptIn(ExperimentalCustomerSheetApi::class)
+@Subcomponent
+internal interface CustomerSheetComponent {
+    val customerSheet: CustomerSheet
+    val stateComponent: CustomerSessionComponent
+
+    @Subcomponent.Builder
+    interface Builder {
+        @BindsInstance
+        fun lifeCycleOwner(lifecycleOwner: LifecycleOwner): Builder
```

Comment:
Do we need this? I think we only need it as part of creating the session view model, and don't need it here.